static-assets/components/cstudio-forms/controls/internal-name.js

```diff
@@ -16,237 +16,241 @@
  */
 
 CStudioForms.Controls.InternalName = CStudioForms.Controls.InternalName ||
-function(id, form, owner, properties, constraints, readonly)  {
-	this.owner = owner;
-	this.owner.registerField(this);
-	this.errors = []; 
-	this.properties = properties;
-	this.constraints = constraints;
-	this.inputEl = null;
-	this.patternErrEl = null;
-	this.countEl = null;
-	this.required = true;
-	this.value = "_not-set";
-	this.form = form;
-	this.id = "internal-name";
-	this.readonly = readonly;
-	
-	return this;
-}
+  function (id, form, owner, properties, constraints, readonly) {
+    this.owner = owner;
+    this.owner.registerField(this);
+    this.errors = [];
+    this.properties = properties;
+    this.constraints = constraints;
+    this.inputEl = null;
+    this.patternErrEl = null;
+    this.countEl = null;
+    this.required = true;
+    this.value = "_not-set";
+    this.form = form;
+    this.id = "internal-name";
+    this.readonly = readonly;
+    this.i18n = CrafterCMSNext.i18n;
```

Comment:
Let’s stop using this pattern. Best to create closures and avoid `this` resetting on the i18n related libs.